api/API_VERSIONING.md

```diff
@@ -144,6 +151,18 @@ only be made to the *current stable major version*. The rationale for this polic
 * We encourage Envoy users to move to the current stable major version from the previous one to
   consume new functionality.
 
+As previously discussed, the new feature is added to the current minor version, and increases the
+patch version.
+
+# API feature deprecation
+
+A feature can be deprecated by annotating the corresponding field or enum value as `[deprecated = true]`.
+This implies that the feature is deprecated during the current minor version, `Y`.
+When the minor version is cut (starting with minor version `Y+1`), automatic tools will add another annotation,
+`[deprecated = true, minor_ver=Y]` (or `minor_ver_enum` for an enum value), indicating that the field
+(or enum value) were deprecated during minor version Y. 2 years later, when minor version `Y+2` will
```

Comment:
1-2 years later when minor version Y+1..

Reply:
Rephrased and updated.